### ISSUE TITLE
Page title service (2.4.2)

### ISSUE THREAD

**Issue by Blackbaud-ToddRoberts** [blackbaud/skyux-core#67]
From @Blackbaud-MattGregg SPA navigation guideline work.

Automatically construct appropriate page titles based on product/feature. 
Automatically announce page title in screen readers.

**Comment by Blackbaud-PaulCrowder** [comment]
Helpful info on how to handle dynamic titles in a SPA: https://hiddedevries.nl/en/blog/2018-07-19-accessible-page-titles-in-a-single-page-app

**Comment by Blackbaud-BobbyEarl** [comment]
Pointing out that https://developer.blackbaud.com/skyux/design/guidelines/page-layouts#page-titles may need to be updated depending on the outcome of this issue.

**Comment by Blackbaud-PaulCrowder** [comment]
Addressed in #113
